### DISCUSSION TITLE
RM551E-GL RNDIS to MBIM mode

### DISCUSSION THREAD

**Discussion by Zigmabeta** [iamromulan/cellular-modem-wiki#10] [Help - Quectel]
Hi, I changed my modem to USBNET=3 and lost AT command port. Is there anyway to revert back to USBNET=2 or 0?

https://ibb.co/xFcRsxh

**Comment by kevinhorner** [comment]
Its been over a year since you posted but I am commenting in case someone else finds this thread.
I connected mine to a windows machine and while the serial ports weren't automatically recognized I could see them in device manager and could manually install a quectel usb serial port driver. I did get a warning about the driver not being compatible but it still worked to give me the ability to run AT commands.

**Comment by dr-jimmy261** [comment]
just install RNDIS driver and then reboot, that's fixed for me :D 
![image](https://github.com/user-attachments/assets/81d123b2-62d9-4e7b-a4af-20802066599e)
